pkg/cli/config/config.go

```diff
@@ -1,35 +1,21 @@
 package config
 
 import (
-	"encoding/json"
 	"fmt"
 	"os"
 	"path/filepath"
 
 	"github.com/openshift/backplane-cli/pkg/info"
+	"github.com/spf13/viper"
 )
 
 type BackplaneConfiguration struct {
-	URL      string `json:"url"`
-	ProxyURL string `json:"proxy-url"`
+	URL      string `mapstructure:"url"`
```

Comment:
Could you help me understand why this?

Reply:
From the [documentation](https://github.com/spf13/viper#unmarshaling):

> Viper uses [github.com/mitchellh/mapstructure](https://github.com/mitchellh/mapstructure) under the hood for unmarshaling values which uses mapstructure tags by default.